.github/workflows/dbld-images.yml

```diff
@@ -0,0 +1,57 @@
+name: Compile dbld-images
+
+on:
+  pull_request:
+  push:
+    paths:
+      - .github/workflows/dbld-images.yml
+      # List is from dbld/rules cache-image-% "WATCHED_FILES"
+      - "dbld/**"
+      - "packaging/rhel/syslog-ng.spec"
+      - "packaging/debian/control"
+  schedule:
+    - cron: '00 03 * * *'
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+  
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Build the images
+        run: dbld/rules images image-devshell -j $(nproc)
+
+      - name: Should we upload the images?
+        run: |
+          if [ "${{ github.repository_owner }}" = "syslog-ng" ] && \
+             [ "${{ github.ref }}" = "refs/heads/master" ] && \
+             [ "${{ github.event_name }}" != "schedule" ]
```

Comment:
Shouldn't we be strict here so a change/refactor on the top of this file will not break anything ever?

I mean if I understand correctly, our registry push trigger is simple: we want that only "when something new is _pushed_ into the _master_ branch of the _syslog-ng_ upstream repository".

Reply:
My intention was to allow "manual image update" with `workflow_dispatch`.
However we never did it with DockerHub either. So I removed that part, and this logic is reduced to "if push".

Thank You for the comment!